scripts/mh_sbuserinfo.pl

```diff
@@ -0,0 +1,344 @@
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL
+# THE AUTHOR BE LIABLE FOR  ANY SPECIAL, DIRECT, INDIRECT, OR
+# CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+# LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
+# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+# CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+##############################################################################
+#
+# statusbar item that shows userinfo in channels
+#
+# displays in the statusbar the number of users and the limit of the channel,
+# with several settings for finetuning:
+#
+# default settings: [<users>(<users_op>:<users_voice>:<users_rest>)/<limit>]
+# "/<limit>" will only show when there is a limit set
+#
+# setting mh_sbuserinfo_show_details (default ON): enable/disable showing a
+# detailed breakout of users into ops, halfops, voice and normal
+#
+# setting mh_sbuserinfo_show_details_mode (default OFF): enable/disable
+# prefixing ops, halfops and voice with @%+ when details are enabled
+#
+# setting mh_sbuserinfo_show_details_halfop (default OFF): enable/disable
+# showing halfops when details are enabled
+#
+# setting mh_sbuserinfo_show_warning_opless' (default ON): change the colour
+# of "<users_op>" if channel is opless
+#
+# setting mh_sbuserinfo_show_warning_limit (default ON): change the colour
+# of "<limit>" if channel is above, at or close to the limited amount of users
+#
+# setting mh_sbuserinfo_show_warning_limit_percent (default 90): number in
+# percent (0-100) of users relative to the limit before a limit warning is
+# triggered
+#
+# setting mh_sbuserinfo_warning_format (default '%Y'): the colour used for
+# warnings. see http://www.irssi.org/documentation/formats
+#
+# to configure irssi to show the new statusbar item in a default irssi
+# installation type '/statusbar window add -after window_empty mh_sbuserinfo'.
+# see '/help statusbar' for more details and do not forget to '/save'
+#
+# history:
+#	v1.00 (20151126)
+#		initial release
+#
+
+use v5.14.2;
+
+use strict;
+
+##############################################################################
+#
+# irssi head
+#
+##############################################################################
+
+use Irssi 20100403;
+use Irssi::TextUI;
+
+our $VERSION = '1.00';
+our %IRSSI   =
+(
+	'name'        => 'mh_sbuserinfo',
+	'description' => 'statusbar item that shows userinfo in channels',
+	'license'     => 'BSD',
+	'authors'     => 'Michael Hansen',
+	'contact'     => 'mh on IRCnet #help',
+	'url'         => 'http://scripts.irssi.org',
+);
+
+##############################################################################
+#
+# script functions
+#
+##############################################################################
+
+sub statusbar_redraw
+{
+	my ($channel) = @_;
+
+	if (ref($channel) eq 'Irssi::Irc::Channel')
```

Comment:
There are for example SILC or XMPP channels. Users, ops and limits are a general concept. But you're also right, I am not sure if they would have the synced flag. Given that you can't be expected to go through the trouble of testing your script on all other channels maybe it's better to leave it specific.